diff -urN jed-B0.99-17.111.org/autoconf/aclocal.m4 jed-B0.99-17.111/autoconf/aclocal.m4
--- jed-B0.99-17.111.org/autoconf/aclocal.m4	2005-07-25 06:37:22.000000000 +0200
+++ jed-B0.99-17.111/autoconf/aclocal.m4	2005-09-16 09:55:30.489122128 +0200
@@ -207,11 +207,7 @@
 # the other directories may have a better chance of containing a more recent
 # version.
 
-JD_Search_Dirs="$JD_Search_Dirs \
-                /usr/local/include,/usr/local/lib \
-		/usr/include,/usr/lib \
-		/usr/include/$1,/usr/lib \
-		/usr/include/$1,/usr/lib/$1"
+JD_Search_Dirs="/usr/include/$1,$libdir"
 
 echo looking for the JD_UP_NAME library
 
@@ -221,21 +217,13 @@
   generic_include=`echo $include_and_lib | tr ',' ' ' | awk '{print [$]1}'`
   generic_lib=`echo $include_and_lib | tr ',' ' ' | awk '{print [$]2}'`
   echo Looking for $1.h in $generic_include
-  echo and lib$1.a in $generic_lib
-  if test -r $generic_include/$1.h && test -r $generic_lib/lib$1.a
+  echo and lib$1.so in $generic_lib
+  if test -r $generic_include/$1.h && test -r $generic_lib/lib$1.so
   then
     echo Found it.
     JD_UP_NAME[]_LIB_DIR="$generic_lib"
     JD_UP_NAME[]_INCLUDE="$generic_include"
     break
-  else
-    if test -r $generic_include/$1.h && test -r $generic_lib/lib$1.so
-    then
-      echo Found it.
-      JD_UP_NAME[]_LIB_DIR="$generic_lib"
-      JD_UP_NAME[]_INCLUDE="$generic_include"
-      break
-    fi
   fi
 done
 
