--- jed-0.99-16/src/Makefile.in.orig	2002-10-20 08:55:30.000000000 +0200
+++ jed-0.99-16/src/Makefile.in	2003-11-25 23:48:06.000000000 +0100
@@ -14,9 +14,11 @@
 exec_prefix = @exec_prefix@
 #JED_ROOT = $(HOME)/src/jed# <--- make sure # is at end of file!!
 #JED_ROOT = $(prefix)/jed# <--- make sure # is at end of file!!
-JED_ROOT = @JED_ROOT@# <--- make sure # is at end of file!!
+#JED_ROOT = @JED_ROOT@# <--- make sure # is at end of file!!
+JED_ROOT = @datadir@/jed# <--- make sure # is at end of file!!
 BIN_DIR = @bindir@
 MAN_DIR = @mandir@/man1
+INFO_DIR = @infodir@
 INSTALL		= @INSTALL@
 INSTALL_DATA	= @INSTALL_DATA@
 MKINSDIR = ../autoconf/mkinsdir.sh
@@ -51,10 +53,10 @@
 #---------------------------------------------------------------------------
 # 1. Linux GPM Mouse support
 #    Uncomment next FOUR lines for GPM mouse support
-#MOUSEFLAGS = -DHAS_GPM_MOUSE
-#MOUSELIB = -lgpm
-#GPMMOUSEO = gpmmouse.o
-#OBJGPMMOUSEO = $(OBJDIR)/$(GPMMOUSEO)
+MOUSEFLAGS = -DHAS_GPM_MOUSE
+MOUSELIB = -lgpm
+GPMMOUSEO = gpmmouse.o
+OBJGPMMOUSEO = $(OBJDIR)/$(GPMMOUSEO)
 
  
 # 2. XFree86 XRENDERFONT (Anti-aliased font) support for XJED
