--- jed-0.99-19/src/Makefile.in.orig	2016-06-26 14:08:11.521834233 +0200
+++ jed-0.99-19/src/Makefile.in	2016-06-26 14:17:35.022114822 +0200
@@ -15,7 +15,7 @@
 datarootdir = @datarootdir@
 #JED_ROOT = $(HOME)/src/jed# <--- make sure # is at end of file!!
 #JED_ROOT = $(prefix)/jed# <--- make sure # is at end of file!!
-JED_ROOT = @JED_ROOT@# <--- make sure # is at end of file!!
+JED_ROOT = @datadir@/jed# <--- make sure # is at end of file!!
 BIN_DIR = @bindir@
 MAN_DIR = @mandir@/man1
 INFO_DIR = @infodir@
