--- ./src/Makefile.in.orig	Fri Jul 13 20:36:44 2001
+++ ./src/Makefile.in	Fri Jul 13 20:51:44 2001
@@ -14,9 +14,11 @@
 exec_prefix = @exec_prefix@
 #JED_ROOT = $(HOME)/src/jed# <--- make sure # is at end of file!!
 #JED_ROOT = $(prefix)/jed# <--- make sure # is at end of file!!
-JED_ROOT = @JED_ROOT@# <--- make sure # is at end of file!!
+#JED_ROOT = @JED_ROOT@# <--- make sure # is at end of file!!
+JED_ROOT = @datadir@/jed# <--- make sure # is at end of file!!
 BIN_DIR = @bindir@
 MAN_DIR = @mandir@/man1
+INFO_DIR = @infodir@
 INSTALL		= @INSTALL@
 INSTALL_DATA	= @INSTALL_DATA@
 MKINSDIR = ../autoconf/mkinsdir.sh
@@ -28,7 +30,7 @@
 JED_COLOR_FILES = ../lib/colors/*.sl ../lib/colors/README
 XJED_COLOR_FILES = ../lib/colors/Xjed/*.sl
 JED_TM_FILES = ../lib/tm/*.tm 
-JED_INFO_FILES = ../info/*
+JED_INFO_FILES = ../info/jed.*
 
 #---------------------------------------------------------------------------
 # DESTDIR is designed to facilitate making packages.  Normally it is empty
@@ -52,10 +54,10 @@
 #---------------------------------------------------------------------------
 # 1. Linux GPM Mouse support
 #    Uncomment next FOUR lines for GPM mouse support
-#MOUSEFLAGS = -DUSE_GPM_MOUSE
-#MOUSELIB = -lgpm
-#GPMMOUSEO = gpmmouse.o
-#OBJGPMMOUSEO = $(OBJDIR)/$(GPMMOUSEO)
+MOUSEFLAGS = -DUSE_GPM_MOUSE
+MOUSELIB = -lgpm
+GPMMOUSEO = gpmmouse.o
+OBJGPMMOUSEO = $(OBJDIR)/$(GPMMOUSEO)
 
 #---------------------------------------------------------------------------
 # S-Lang library location
